### ISSUE TITLE
Why fc_coord is Linear() Instead of nn.linear in Regression_nf_3d.py?

### ISSUE THREAD

**Issue by 2015wan** [Jeff-sjtu/res-loglikelihood-regression#61]
1. Why design a separate linear layer for the output of the coordinate mean, and what is the difference between it and the ordinary fully connected layer?
2. 
3. Why is this happening?
x_norm = torch.norm(x, dim=1, keepdim=True)
y = y / x_norm

**Comment by 2015wan** [comment]
In this class RegressFlow3D(nn.Module) Why when testing
pred_jts[:, :, 2] = pred_jts[:, :, 2] - pred_jts[:, self.root_idx:self.root_idx + 1, 2]
Why do you have to subtract like this
